build/raise-ivy-projects-version/raise.sh

```diff
@@ -82,6 +82,9 @@ raiseProjects() {
 updateProjectRepos() {
   projectRepos=(
     "git@github.com:axonivy/core.git"
+    "git@github.com:axonivy-market/demo-projects.git"
+    "git@github.com:axonivy-market/doc-factory.git"
+    "git@github.com:axonivy-market/msgraph-connector.git"
   )
```

Comment:
@weissreto the raise-ivy-project build is now able to update all our leading-edge repos at once :rocket: